kit/src/main/java/com/oracle/javafx/scenebuilder/kit/library/util/FolderExplorer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, Gluon and/or its affiliates.
+ * Copyright (c) 2019, 2020, Gluon and/or its affiliates.
```

Comment:
Shouldn't this change be "2017, 2020, Gluon" ?

Reply:
No, that file was actually created in 2019, in this PR https://github.com/gluonhq/scenebuilder/pull/228